### PR TITLE
Ag 57 58 onboarding accessibility fixes

### PR DESCRIPTION
**Pull request by lucaschifino**
[AG-57](https://ecosia.atlassian.net/browse/AG-57)
[AG-58](https://ecosia.atlassian.net/browse/AG-58)

## Context
Follow-up of #474.

There were a couple issues found with the accessibility improvements made on the onboarding.
1) Welcome page still voices over the images in the text as “attachment png“
2) All onboarding pages are using the first title and subtitle, i.e. accessibility title and subtitle not updating

## Approach
1) Use the localized text before adding the image attachments as accessibility label. Also removing the new lines from it so it does not add pauses when voicing over.
2) Move the update of the accessibility labels from `addDynamicViews()` to `display()`, since the first only happens once, while the latter happens every time the content is updated.

## Additional improvents
- Removed the "This is the onboarding title" and "This is the onboarding subtitle" voice over and left only the actual value, since it makes going over the onboarding faster and less repetitive.
- Since we re-use the same screen for all onboarding steps, it was not jumping back to the top after the user changes steps. That's why I added a `UIAccessibility` notification to force it to change to the title after a new step is displayed.

### CODE REVIEW

File: Client/Ecosia/UI/Onboarding/Welcome.swift [hunk 1]
```diff
@@ -138,6 +138,7 @@ final class Welcome: UIViewController {
         let label = UILabel()
         label.numberOfLines = 0
         label.attributedText = introText
+        label.accessibilityLabel = simplestWayString.replacingOccurrences(of: "\n", with: "")
```
Comment (d4r1091) [inline Client/Ecosia/UI/Onboarding/Welcome.swift:141]: 💡

File: Client/Ecosia/UI/Onboarding/WelcomeTour.swift [hunk 2]
```diff
@@ -257,6 +252,16 @@ final class WelcomeTour: UIViewController,  NotificationThemeable {
                 self.view.layoutIfNeeded()
             }
         }
+        
+        updateAccessibilityLabels(step: step)
```
Comment (d4r1091) [inline Client/Ecosia/UI/Onboarding/WelcomeTour.swift:256]: 👌

**Review by d4r1091 — DISMISSED** [review]
Good suggestion on removing the "This is...", especially whether after testing a bit makes you feeling like you're having some friction into experiencing the Onboarding scenes via VoiceOver

File: Client/Ecosia/UI/Onboarding/WelcomeTour.swift [hunk 3]
```diff
@@ -306,6 +308,7 @@ final class WelcomeTour: UIViewController,  NotificationThemeable {
         Analytics.shared.introClick(.next, at: onboardingAnalyticsPageFromCurrentIndex)
         let displayingStep = currentIndex + 1
         display(step: steps[displayingStep])
+        UIAccessibility.post(notification: .screenChanged, argument: titleLabel)
```
Comment (d4r1091) [inline Client/Ecosia/UI/Onboarding/WelcomeTour.swift:311]: noice!

**Review by d4r1091 — APPROVED** [review]
Going to approve it again 😉 , nice improvements 🙌

**Review by ecotopian — APPROVED** [review]
Looks very accessible now :accessibility:

**Pull request merged**